src/display/annotation_layer.js

```diff
@@ -2699,20 +2702,22 @@ class AnnotationLayer {
    * @param {AnnotationLayerParameters} params
    * @memberof AnnotationLayer
    */
-  static update(params) {
-    const { annotationCanvasMap, div, viewport } = params;
-    setLayerDimensions(div, { rotation: viewport.rotation });
+  update(params) {
+    const { viewport } = params;
```

Comment:
Nit: Perhaps just simplify this as follows?
```suggestion
  update({ viewport }) {
```